server/amqp_request.go

```diff
@@ -88,6 +92,10 @@ func (r *AMQPRequest) Interface() interface{} {
 	return r.delivery
 }
 
+func (r *AMQPRequest) AccessToken() string {
+	return r.accessToken
+}
+
 // Client implementation
 
 // ScopedRequest calls a service within the scope of the current request
```

Comment:
I think it may be worth passing the access token to the child request even in this PR. Otherwise we can only play around with authentication on the "edge" services that get hit by the API